### ISSUE TITLE
"Missing Textures" Crash?

### ISSUE THREAD

**Issue by WildSpirits** [jadedcat/AS2Configs#232]
Hi,

I've been playing Agrarian Skies 2 with a friend. We've added 3 mods into the pack, those being Decocraft, Chisel 3, and Dragon Mounts. However, after a while I have started crashing at seemingly random times. It's gotten to the point where, as soon as I connect to the server, Minecraft crashes. I've even tried to play in single player, and I crashed while going through my inventory.

All I can seem to figure out from my log is that I'm missing some textures, but that shouldn't be the problem since I've redownloaded the problem mods and saw that they lacked the files, too. My friend and I are at a bit of a loss, so help would be greatly appreciated!

Here's the pastebin:
http://pastebin.com/n8mi4P8X

**Comment by Trisscar05** [comment]
I can't seem to find the actual crash in there. Is this the entire log?

**Comment by WildSpirits** [comment]
Hmm, perhaps not. I just copied what was available in the launcher. Pastebin won't let me upload the entire log file from my latest crash. I hope google drive is all right.

https://drive.google.com/file/d/0BzbKUsvyoSQ2OWNDZXQ4ZkltLW8/view?usp=sharing

**Comment by Trisscar05** [comment]
Wait.... Chisel was already on the pack as far as I know. Did you take out the old version when you put in the new one? I ask because there are a lot of "already registered" lines for chisel blocks.
Also, which version of Decocraft, the one with the infinite recipes of different placement of clay, or the newer one that I think has a workbench for it all now?

**Comment by WildSpirits** [comment]
Nope, Chisel isn't included by default. I even tried to remove Chisel and it still crashed. I would like to point out that my friend can play fine for some reason.

As for Decocraft, we have the one with the workbench.

**Comment by Trisscar05** [comment]
Huh, mk. Couple things to find our bearings then:
-The crash log for single player if you would, since the log for the server play is unlikely to show anything unless it's from the host computer, and the single player one will likely be easier to solve first.
-Basic PC specs, ie OS (brand bit and version), total and allowed-to-the-pack-now RAM (this is found by going to options in the Curse or FTB launcher), graphics card, etc.
-Crash log from the server host if you can get it, since that'll have more info than a log from someone just joining it.

**Comment by WildSpirits** [comment]
So it's decided not to crash in single player anymore, at least from some quick testing. I _think_ that this is the log from the last time it crashed, though, which doesn't really look all that helpful to me:
http://pastebin.com/fVCbCB9D

I have Windows 8.1 64 bit, 8gb ram (with up at 6gb for Minecraft via Curse, it recommends 4gb), and all I have graphics-wise is an Intel(R) HD Graphics Family chipset, haha.

Here's the server log from the latest crash:
https://drive.google.com/file/d/0BzbKUsvyoSQ2RXEtWTNINWhBSlE/view?usp=sharing

I'm not sure it's my specs, since when my friend started a new world I could connect fine. He even tried deleting my player data, and I managed to connect. However, when I flew back to our base, almost as soon as it came into view it crashed.

**Comment by pjfranke** [comment]
> I have Windows 8.1 64 bit, 8gb ram (with up at 6gb for Minecraft via Curse, it recommends 4gb), and all I have graphics-wise is an Intel(R) HD Graphics Family chipset, haha.

Not that it will fix your issue, but that much is unnecessary.  Try running at 3-3.5GB of RAM allocation, and without any texture packs.  I consistently play various packs with only 3-3.5 allocated, and don't have memory issues in the game.

Intel HD graphics aren't great, but shouldn't be the issue if you've found them sufficient so far.

**Comment by Trisscar05** [comment]
Ugh, why is Agricraft so techy about everything....
Anyway, do you have an idea of what the time was when you crashed in the server log?

**Comment by WildSpirits** [comment]
About 13:20, I believe.

> [13:20:55] [Server thread/DEBUG] [NotEnoughItems/]: Unloading Player: WildSpirits

I also narrowed down my client log to when I crashed, too, if you want that (ignore the different timestamps):
http://pastebin.com/AJP3vTF3

**Comment by Trisscar05** [comment]
[13:20:55] [Netty IO #7/ERROR] [FML/]: NetworkDispatcher exception
io.netty.handler.timeout.ReadTimeoutException
^Above is the only line that tries to explain why you were booted. So a couple things now:
What's your Java version and what is the hosts'?
How far along are you all in the pack? It should give some idea as to any blocks that might be causing the issues.
Have the host grab a copy of MCEdit and load the world into it. This is a long shot, since they aren't crashing themselves, but it's a fast way to check if any chunks are corrupted, in which case the chunks might be completely black instead of showing what's in them.

**Comment by WildSpirits** [comment]
Both of us have Java 8.
I started crashing about... maybe when we got the barrels? It wasn't too far in.
Nope, he looked and no luck.

We restarted the world from scratch, and it stopped crashing. We made it quite a ways in-- well into the first chapter, and a few quests into the tinkering chapter. We have an automatic cobble gen, semi-auto composter, etc. All I did was open the crafting table (which I've done many, many times before) and it's acting up again. :(
